content/platform/data-services-cloud-console-api/home.md

```diff
@@ -0,0 +1,133 @@
+
+### Documented Attributes
+
+* **API Name & Description** - Provide short description of the goal for this API with the supported HTTP request method (POST, GET, etc)
+* **API Path** - Provide detail URL path as the end-point to issue the API call
+* **API Parameter**s - 
+
+### Supported API Categories (Services)
+
+The API categories for DSCC grows in accordance to the expansion of the DSCC services that are going to released in future. Current services that are available as recorded today is:
+
+1. DataOps Manager
+2. Event Audit
+3. Task tracking
+4. Search
+5. Helps, Issues and announcement
+
+### Versioning
+
+The major version number will be provided in the resource path as v1 in this example:
+
+```md
+/api/v1/<resource group>/...
+```
+
+Some examples of these resource paths:
+
+```markdown
+/api/v1/storage-systems/...
+
+/api/v1/controllers/...
+
+/api/v1/volumes/...
+```
+
+Some examples of the resource groups under the same root:
+
+Clients will be able to adopt the backward compatibility from the higher major version incremental.  However, both the new and old version of API will be supported until the announcement of the deprecation. Nonetheless, the older major version will always be frozen with exception of bug fixes. There will also be announcement of deprecation in the header and sunset header. 
+
+```md
+/api/v1/<resource group>/...
+
+/api/v2/<resource group>/...
+```
+
+### HTTP Request Methods
+
+| HTTP Verbs | Description                                                         |
+| ---------- | ------------------------------------------------------------------- |
+| GET        | Retrieves target resource                                           |
+| POST       | Creates an entity or changes state                                  |
+| PUT        | Replaces target resource with data part of the HTTP Request payload |
+| DELETE     | Remove the target source                                            |
+
+### Relationship Authorization
+
+The client will receive only the properties that are authorized based on the Role Base Access Control for the user who created the access token. The authorization for the client will inherit the user's permission who created the Client Application registration under the API Gateway. Note that subsequent change to the user's permission after the Client Application registered will impact the response returned based on current authority.
+
+### Asynchronous Response
+
+All of the REST API operations are stateless in nature, such as POST, in that scenario the task resource will return a response with HTTP code 202 "Accepted" and the reference to the task as follows:
+
+```md
+Response: 202 Accepted
+```
+
+```md
+{
+   "taskURi": "/api/v1/tasks/{task id}"
+}
+```
+
+In order to ensure the completion of this remote procedural call through POST, user will use the task resource to query the status of this asynchronous task.
```

Comment:
Either change "user" to "users" or add in the word "the" prior to "user" (honestly, I'd just make it plural, in keeping with the rest of the article)